src/NServiceBus.AcceptanceTesting/ScenarioContext.cs

```diff
@@ -7,6 +7,8 @@
 
     public abstract class ScenarioContext
     {
+        public Guid UniqueTestRunId { get; } = Guid.NewGuid();
```

Comment:
Nitpick: Not sure what the "Unique" prefix adds, `public Guid TestRunId` seems pretty revealing?